frontend/src/pages/Projects/Project/ProjectDetails/Deliverables/index.tsx

```diff
@@ -6,7 +6,7 @@ import { useRenderTableCell } from "components/PLAYGROUND/hooks/useRenderTableCe
 import { useFormControls } from "hooks";
 import { useFormData } from "hooks/useFormData";
 import { IFormControls } from "types";
-import { tableConfig } from "./TableConfig";
+import { tableConfig } from "./tableConfig";
```

Comment:
it seems like this keeps getting toggled back and forth, i swear in another pr it was ./TableConfig

Reply:
yes I swear someone is messing with me....

Reply:
Maybe a difference between your and @robrien-bcgov's linter versions or something?